app/scripts/services/Session.svc.js

```diff
@@ -24,11 +26,33 @@ angular.module('yourStlCourts').factory('Session', function () {
     lastSelectedCitation = null;
   }
 
+  function setLatestDOB(dob){
+    latestDOB = formatDOB(dob);
+  }
+
+  function getLatestDOB(){
```

Comment:
Why are we storing the DOB here? It's only used in 1 place which is the same place it's set from. I think we can get rid of all the DOB stuff in this Session service.....